backend/src/controllers/paymentController.ts

```
@@ -0,0 +1,565 @@
+        success: false,
+        error: 'Authentication required',
+      });
+      return;
+    }
+
+    const { status, page = '1', limit = '10' } = req.query;
+    const pageNum = parseInt(page as string);
+    const limitNum = parseInt(limit as string);
+    const skip = (pageNum - 1) * limitNum;
+
+    let where: any = {};
```

ESLint

[error] 'where' is never reassigned. Use 'const' instead.

(prefer-const)